components/script/dom/htmlmediaelement.rs

```diff
@@ -2454,6 +2452,8 @@ impl VirtualMethods for HTMLMediaElement {
     fn unbind_from_tree(&self, context: &UnbindContext) {
         self.super_type().unwrap().unbind_from_tree(context);
 
+        self.remove_controls();
+
```

Comment:
Hrm. I was curious why we could do this, but not need to re-register the controls when binding to the tree, but it looks like controls are registered every time they render! That's a bit odd, but I don't see an issue with this until that changes.